### PR TITLE
Always connect the uniformity graph after a function call.

### PR DESCRIPTION
**Pull request by alan-baker**
For function calls, if the call was tagged as CallSiteRequiredToBeUniform.S (any severity) then no edge was added to link the before (last CF_i) and after (CF_after) the call. This was ok when the severity was always an error, but if the severity is warning or info, the rules will miss a collective operation requirement after the function call at a higher severity. It should always be safe to connect the control flow before and after. The disconnection was simply an optimization when diagnostics were always errors.

### CODE REVIEW

**Review by jrprice — APPROVED** [review]
LGTM, matches what Tint already does.

File: wgsl/index.bs [hunk 1]
```diff
@@ -10785,8 +10785,6 @@ The most complex rule is for function calls:
 
 Note: Refer to [[#func-var-value-analysis]] for the definition of *Vout*(*call*).
 
-A [=value constructor=] expression is analyzed as if it were a function call with the same arguments.
```
Comment (dneto0) [inline wgsl/index.bs:10788]: This is unrelated.

**Review by dneto0 — CHANGES_REQUESTED** [review]
Unrelated change was lumped in with this.  ??

File: wgsl/index.bs (hunk 1 above)
Comment (alan-baker) [reply wgsl/index.bs:10788]: Yes it is an editorial only change since we moved the value constructors to built-in functions. See the 2nd commit message.

**Review by dneto0 — APPROVED** [review]
Thanks!
